Games/types/Mafia/roles/cards/WinWithMafia.js

```diff
@@ -8,13 +8,13 @@ module.exports = class WinWithMafia extends Card {
     this.winCheck = {
       priority: PRIORITY_WIN_CHECK_DEFAULT,
       check: function (counts, winners, aliveCount) {
-        if (counts["Mafia"] >= aliveCount / 2 && aliveCount > 0)
-          winners.addPlayer(
-            this.player,
-            this.player.role.alignment == "Mafia"
-              ? "Mafia"
-              : this.player.role.name
-          );
+        if (counts["Mafia"] >= aliveCount / 2 && aliveCount > 0 ||
+        (this.game.players.filter(e => e.role.name === "Dignitary").length > 0 &&
```

Comment:
i will add a separate card to force mafia wins

Reply:
so that games without this role (which is quite a large majority of games) will not unnecessarily run this function